easybuild/easyconfigs/g/gmvolf/gmvolf-18.05.eb

```diff
@@ -0,0 +1,35 @@
+# contributed by Luca Marsella(CSCS)
+
+easyblock = "Toolchain"
+name = 'gmvolf'
+version = '18.05'
+
+homepage = '(none)'
+description = """GNU Compiler Collection (GCC) based compiler toolchain, including
+ MVAPICH2 for MPI support, OpenBLAS, FFTW and ScaLAPACK."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+compname = 'GCC'
+compver = '8.1.0-2.30'
+comp = (compname, compver)
+
+blaslib = 'OpenBLAS'
+blasver = '0.2.20'
+blas = '%s-%s' % (blaslib, blasver)
+
+# toolchain used to build foss dependencies
+comp_mpi_tc_name = 'gmvapich2'
+comp_mpi_tc = (comp_mpi_tc_name, version)
+
+# compiler toolchain depencies
+# we need GCC and MVAPICH2 as explicit dependencies instead of gmvapivh2 toolchain
+dependencies = [
+    comp,
+    ('MVAPICH2', '2.3rc2', '', ('GCC', '8.1.0-2.30') ),
```

Comment:
OK but remember that the installation path of this toolchain should be prefixed by `experimenal`